src/document/position.js

```diff
@@ -113,11 +135,21 @@ CKEDITOR.define( [ 'utils', 'ckeditorerror' ], ( utils, CKEditorError ) => {
 		 *
 		 * @returns {Number[]} Path to the parent.
 		 */
-		getParentPath() {
```

Comment:
We should decide something here: https://github.com/ckeditor/ckeditor5-design/issues/88.